torch/csrc/distributed/autograd/rpc_messages/cleanup_autograd_context_req.cpp

```diff
@@ -0,0 +1,46 @@
+#include <torch/csrc/distributed/autograd/rpc_messages/cleanup_autograd_context_req.h>
+#include <torch/csrc/jit/pickle.h>
+
+namespace torch {
+namespace distributed {
+namespace autograd {
+
+CleanupAutogradContextReq::CleanupAutogradContextReq(int64_t context_id)
+    : context_id_(context_id){};
+
+int64_t CleanupAutogradContextReq::getContextId() {
+  return context_id_;
+}
+
+rpc::Message CleanupAutogradContextReq::toMessage() && {
+  std::vector<at::IValue> ivalues;
+  ivalues.emplace_back(context_id_);
```

Comment:
nit: `std::vector<at::IValue> ivalues {context_id_};`

Reply:
No longer needed per your suggestion since we just have a single value we are pickling.